packages/fxa-settings/src/components/Settings/FlowRecoveryKeyDownload/index.stories.tsx

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import FlowRecoveryKeyDownload from '.';
+import { Meta } from '@storybook/react';
+import { useFtlMsgResolver } from '../../../models';
+import { withLocalization } from '../../../../.storybook/decorators';
+
+export default {
+  title: 'Components/Settings/FlowRecoveryKeyDownload',
+  component: FlowRecoveryKeyDownload,
+  decorators: [withLocalization],
+} as Meta;
+
+const currentStep = 3;
+const numberOfSteps = 4;
+const recoveryKeyValue = 'ANMD 1S09 7Y2Y 4EES 02CW BJ6Z PYKP H69F';
```

Comment:
Not sure it matters, but this string is also exported from https://github.com/mozilla/fxa/blob/25f936cc4e80e09dc640089e03276f175415d2d6/packages/fxa-settings/src/components/Settings/FlowRecoveryKeyDownload/mocks.tsx#L5-L6